Projects/Home/Sources/Viewables/ActiveSection.swift

```diff
@@ -15,14 +16,12 @@ extension ActiveSection: Presentable {
         let bag = DisposeBag()
         let section = SectionView()
 
-        let store: HomeStore = self.get()
-
         section.dynamicStyle = .brandGrouped(
             insets: .init(top: 0, left: 14, bottom: 0, right: 14),
             separatorType: .none
         )
 
-        let claims = ClaimSectionLoading()
+        let claims = Claims()
```

Comment:
@sampettersson To display the claim cards stuff on home, I had to import Claims here.
Do you think there is a way by which I can bypass this import and call it via Ugglan instead?

Reply:
@SatishBV Make home take a generic or just an AnyView, and then passing it in from ugglan should work!